docs/endpoints/user.md

```diff
@@ -12,33 +14,100 @@ Create a new user.
 ```
 
 #### Body
-The body is a json format of a user.
+The body is a json format of a user, with an embedded reference to the Account the user will be linked to.
+Optionally, the Account (and underlying Network and Instance) can be created in one request.
```

Comment:
Good point! It's undecided at the moment. I would say that only users that are associated with that specific network can create new users associated with that network.
What do you think?